pkg/cmd/server/api/validation/master.go

```diff
@@ -192,5 +192,12 @@ func ValidatePolicyConfig(config api.PolicyConfig) fielderrors.ValidationErrorLi
 func ValidateProjectRequestConfig(config api.ProjectRequestConfig) fielderrors.ValidationErrorList {
 	allErrs := fielderrors.ValidationErrorList{}
 
+	if len(config.ProjectRequestTemplate) > 0 {
```

Comment:
> The sentence following that is misleading :) 'If it is not specified, then projectrequest will not work.'

I'm open to other verbiage.  The intent is to allow an administrator to easily disable unprivileged project creation by his users.  He could do it via policy, but this seemed pretty clear as well.  Otherwise, he'll have to have something configured that he never intends to use.